client-src/transpiled-modules/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const path = require('path');
+const merge = require('webpack-merge');
+
+const base = {
+  mode: 'production',
+  output: {
+    path: path.resolve(__dirname, '../../client/transpiled-modules'),
+    libraryTarget: 'commonjs2',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: [
+          {
+            loader: 'babel-loader',
+          },
+        ],
+      },
+    ],
+  },
+};
+
+module.exports = [
+  merge(base, {
+    entry: path.join(__dirname, 'log.js'),
+    output: {
+      filename: 'log.js',
+    },
+  }),
+  merge(base, {
+    entry: path.join(__dirname, 'strip-ansi.js'),
+    output: {
+      filename: 'strip-ansi.js',
+    },
+  }),
+];
```

Comment:
I am afraid, some modules can use new syntax (ES 6) and we can't catch it, for example in patch version

Reply:
@evilebottnawi If that happens when we are updating a dependency, it will be caught in CI here: https://github.com/webpack/webpack-dev-server/pull/2658/files#diff-24488ac612ba9666465a7a596040ac73R43

Reply:
But if we want full safety, we should do this: https://github.com/webpack/webpack-dev-server/pull/2652. I don't like how that PR ended up because it has so many breaking changes in the API and seems a little risky

Reply:
Sound good to me. CI is enough for me.